16_bg_slider/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.0-2/css/all.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Slider</title>
+  </head>
+  <body>
+    <div class="box">
+      <div class="slides">
+        <img src="images/slide1.jpeg" style="width: 100%;" class="slides" />
```

Comment:
So, the first thing you need to make the current image visible. A suggestion: Add `.isActive` to the first image by default, that sets `display:block;`. Then you toggle that class through the different images when the next/prev buttons are clicked.